templates/options/boolean.twig

```diff
@@ -0,0 +1,4 @@
+<input type="checkbox" name="{{ option.slug }}" {{ option.value == true ? 'checked' : '' }}><br/>
```

Comment:
i think `option.value === true` is possible here

Reply:
no, not in twig. there it is called is same as.

Reply:
can we use `option.value is same as(true)`?

Reply:
not easily, i need to typecast when getting the value from the database. But it's a good point.